org.scala-ide.sbt.core/pom.xml

```diff
@@ -0,0 +1,182 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.scala-ide</groupId>
+    <artifactId>org.scala-ide.sdt.build</artifactId>
+    <version>4.5.0-SNAPSHOT</version>
+    <relativePath>../org.scala-ide.sdt.build/pom.xml</relativePath>
+  </parent>
+  <artifactId>org.scala-ide.sbt.core</artifactId>
+  <packaging>eclipse-plugin</packaging>
+
+  <properties>
+    <sbtrc.version>0.3.5</sbtrc.version>
+    <pickling.version>0.10.0</pickling.version>
+    <akka.version>2.4.2</akka.version>
+    <reactive.stream.version>1.0.0</reactive.stream.version>
+    <config.version>1.2.1</config.version>
+    <parser.combinators.version>1.0.2</parser.combinators.version>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>com.typesafe.sbt</groupId>
+      <artifactId>client-all-2-11</artifactId>
+      <version>${sbtrc.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.scala-lang.modules</groupId>
+      <artifactId>scala-pickling_2.11</artifactId>
+      <version>${pickling.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>com.typesafe.akka</groupId>
+      <artifactId>akka-stream_2.11</artifactId>
+      <version>${akka.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>com.typesafe.akka</groupId>
+      <artifactId>akka-actor_2.11</artifactId>
```

Comment:
Shouldn't this be `akka-actor_${scala.binary.version}`? We may need to build the IDE for 2.12 soon.